### PR TITLE
Add step to aeotec thermostat setpoint; remove unneeded capabilities

### PR DESCRIPTION
**Pull request by ctowns**
https://smartthings.atlassian.net/browse/CHAD-8786

### CODE REVIEW

File: drivers/SmartThings/zwave-thermostat/profiles/aeotec-radiator-thermostat.yml
Comment (ctowns) [inline drivers/SmartThings/zwave-thermostat/profiles/aeotec-radiator-thermostat.yml:17]: I removed these capabilities because I think they were added by accident at some point. This device doesn't have a cooling set point, humidity, etc.

These capabilities now match the base-radiator-thermostat profile, but the embedded configs are different

**Review by greens — APPROVED** [review]
Has "step" been implemented yet?

**Comment by ctowns** [comment]
Yes, I believe it is now in acceptance! https://smartthings.atlassian.net/browse/CODA-2561

**Comment by greens** [comment]
@ctowns Nice! I'd still wait until you hear it's on Prod to merge this, though.

**Comment by ctowns** [comment]
Sounds good! I'll loop back with Derek and get this in once I've confirmed it's in prod 👍

**Comment by lelandblue** [comment]
I have added the do not merge label until the CODA dependency is in production (expected next Tues)

**Comment by ctowns** [comment]
Just an update: there were some issues with Dossier, so the [necessary CODA change](https://smartthings.atlassian.net/browse/CODA-2561) has not made it to production. I will update as I hear more, but I believe at the moment, this ticket is still waiting to get in acceptance.

**Pull request merged**
